addons/analytic/models/analytic_account.py

```diff
@@ -136,9 +136,9 @@ def _compute_debit_credit_balance(self):
     # use auto_join to speed up name_search call
     partner_id = fields.Many2one('res.partner', string='Customer', auto_join=True, tracking=True, check_company=True)
 
-    balance = fields.Monetary(compute='_compute_debit_credit_balance', string='Balance',  groups='account.group_account_readonly')
-    debit = fields.Monetary(compute='_compute_debit_credit_balance', string='Debit', groups='account.group_account_readonly')
-    credit = fields.Monetary(compute='_compute_debit_credit_balance', string='Credit', groups='account.group_account_readonly')
+    balance = fields.Monetary(compute='_compute_debit_credit_balance', string='Balance')
+    debit = fields.Monetary(compute='_compute_debit_credit_balance', string='Debit')
+    credit = fields.Monetary(compute='_compute_debit_credit_balance', string='Credit')
```

Comment:
So, users with `analytic` and not `account` will display these fields to everyone, is that expected?

Reply:
Well, these fields are currently never shown on views outside of account. (In project, for example, you can access the form view, but the fields are not part of this view, and you can never access the list view).
Balance is but it is something that we want. It should be shown (which is why I removed the group even in account).

I could put debit and credit in invisible in the view in analytic if you prefer, as we don't have an other group to put, but know that it's only necessary if someone did add the view with Studio currently (and I will need to override it in account).